mock/py/mock.py

```diff
@@ -644,7 +894,8 @@ def main():
         uidManager.dropPrivsTemp()
 
     (options, args) = command_parse()
-
+    if rpm is not None:
+        args=rpm
```

Comment:
Originall in args can be only one rpm. You need to run `run_command(options, args, config_opts, commands, buildroot, state)` in a loop.